guide/web-streaming.md

```diff
@@ -15,6 +15,19 @@ except that the `X3D` file and the animation are sent on the fly to the Web brow
 
 %end
 
+The web interface displays a toolbar with the following actions:
+* Simulation time: this item indicates the current simulation time on the first line of the simulation element. The second line is currently not used and indicates how long the simulation will run.
```

Comment:
I would remove "of the simulation element" which I don't understand... About the second line, it sounds strange: it's not used but still indicates something... We should either remove the fact that it is not used or that it indicates something.